lib/aerepl_http/aerepl_utils.ex

```diff
@@ -1,38 +1,31 @@
 defmodule ReplUtils do
+  def render(state, repl_str), do: List.to_string(:aere_repl.render_msg(state, repl_str))
```

Comment:
Better use https://elixirschool.com/en/lessons/basics/pipe-operator/ for such constructs.